helper.js

```diff
@@ -182,6 +181,7 @@ HELPER.compile = (() => {
 	const Sassify = ( scss, css ) => {
 		const compiled = Sass.renderSync({
 			file: scss,
+			indentType: 'tab',
```

Comment:
Uh oh. SPACES! lol. 
Seriously though do you need this setting in combination with `outputStyle: 'compressed'`? will it even have an output?